### DISCUSSION TITLE
Add functionality such as filled variant and copyable from Input to Textarea

### DISCUSSION THREAD

**Discussion by jpangborn** [livewire/flux#264] [Feature Requests]
It would be nice to have some of the functionality from `flux:input` to `flux:textarea`. The two that would be very helpful that I have noticed so far:

- To have the filled variant so that I can have a readonly textarea match a readonly text input.
- To have the ability to use the copyable attribute on the textarea like the input for fields that have code to copy.

**Comment by oMikeo** [comment]
Building on this to save fragmented discussion - would be handy to have access to the same size variants on `<flux:textarea/>` so that  this is possible...

```
<flux:input label="Text" size="sm"/>
<flux:textarea label="Textarea" rows="auto" resize="none" size="sm"/>
```

...and we get a textarea that has an initial height = to the input height.

Textareas would also benefit from min-heights to mitigate this:

<img width="443" alt="Screenshot 2024-11-03 at 18 54 17" src="https://github.com/user-attachments/assets/75701e45-ece8-4bd6-a28f-ce336a5e0264">


```
...
->add(match ($size) {
    default => 'text-sm py-2 min-h-10 leading-[1.375rem]', 
    'sm' => 'text-sm py-1.5 min-h-8 leading-[1.125rem]',
    'xs' => 'text-xs py-1.5 min-h-6 leading-[1.125rem]',
})
...
```

**Comment by haviladev** [comment]
I send this, please add **readonly copyable** features to a textarea, there are cases where the text to show is to big for an input, like showing and needed to be copied big key, an error message, etc.

```
<flux:texarea readonly copyable>
FLUX-1234-5678-ABCD-EFGH-1234-5678-ABCD-EFGH-1234-5678-ABCD-EFGH-1234-5678-ABCD-EFGH-1234-5678-ABCD-EFGH-1234-5678-ABCD-EFGH-1234-5678-ABCD-EFGH"
<flux:textarea>
```

**Comment by wex** [comment]
Agree - I'm using mixed inputs and textareas in "inline editing" solution, where view-only mode uses readonly+filled-variant and textarea... is using disabled to make it even a bit more subtle.
<img width="299" height="206" alt="image" src="https://github.com/user-attachments/assets/15676e20-0411-45ae-b8b1-f704a247c7e1" />
